profile-widget/divecartesianaxis.cpp

```diff
@@ -409,7 +409,7 @@ QColor TimeAxis::colorForValue(double value)
 
 QString TimeAxis::textForValue(double value)
 {
-	int nr = value / 60;
+	int nr = lrint(value / 60);
```

Comment:
nope, I don't think that's correct. We print value % 60 next, so we are doing fraction and reminder...
you don't want 100 seconds to round to 2 minutes and 40 seconds :-)